tutorbackup/templates/backup/build/backup/backup_services.py

```diff
@@ -152,15 +151,14 @@ def archive(paths):
 
 
 def upload_to_s3():
-    from s3_client import S3_CLIENT, IntegrityError
+    from s3_client import S3_CLIENT, IntegrityError, calculate_checksum
 
     bucket = ENV['S3_BUCKET_NAME']
     file_name = TARFILE
 
-    logger.info(f"Uploading {file_name} to S3 bucket {bucket}")
-    calculated_checksum = hashlib.md5(
-        open(file_name, 'rb').read()).hexdigest()
+    calculated_checksum = calculate_checksum(file_name)
 
+    logger.info(f"Uploading {file_name} to S3 bucket {bucket}")
```

Comment:
What I meant in https://github.com/hastexo/tutor-contrib-backup/pull/52#discussion_r976317451 was actually to have two log messages, so as to get an idea of the timing of the checksum calculation. As in:

```python
logger.info(f"Calculating checksum for {file_name}")
calculated_checksum = calculate_checksum(file_name)
logger.info(f"Uploading {file_name} to S3 bucket {bucket}")
try:
    # (etc.)
```

Reply:
Makes sense. I'll add that.